docs/development/pigs/pig-008.rst

```diff
@@ -0,0 +1,449 @@
+.. include:: ../../references.txt
+
+.. _pig-008:
+
+****************
+PIG 8 - Datasets
+****************
+
+* Author: Axel Donath (editor), Christoph Deil, Regis Terrier & Atreyee Sinha
+* Created: Jan 4th, 2018
+* Accepted:
+* Status:
+* Discussion:
+
+
+Introduction
```

Comment:
I see that you put "Introduction" in your PIGs, and I put "Abstract".

Suggest to keep using "Abstract" for consistency.
It is suggested here to put "Abstract":
https://docs.gammapy.org/0.11/development/pigs/pig-001.html

(for this PIG, but also change "Introduction" -> "Abstract" for the roadmap PIG.

Reply:
OK